bin/adde-pre-commit.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+/* eslint-env node */
+
+const path = require('path');
+const lintStaged = require('lint-staged');
+
+(async () => {
+  let lintStagedConfigPath = path.resolve(__dirname, '../.lint-staged.config.js');
+  let success = await lintStaged({
```

Comment:
Do we get any error messages from `lintStaged` that we can surface?

Reply:
Yeah — lint-staged takes over the UI and it prints out the steps it is running and if any failed. I believe that anything that the underlying linters print to stderr gets captured and then shown by lint-staged in its summary.

At any rate: Afaict the same linting-error info is still shown as it was before.